source/administration/minio-console.rst

```diff
@@ -52,9 +52,8 @@ the MinIO Console:
        configuring Prometheus to collect metrics from MinIO.
 
    * - :envvar:`MINIO_SERVER_URL`
-     - The URL hostname the MinIO Console uses for connecting to the MinIO 
-       Server. The hostname *must* be resolveable and reachable for the
-       Console to function correctly.
+     - The URL hostname the MinIO Console uses for connecting to the MinIO Server. 
+       For the Console to function correctly, the MinIO server URL *must* be the full domain name of the host, resolveable, and reachable.
```

Comment:
Maybe we can link to https://en.wikipedia.org/wiki/Fully_qualified_domain_name ?  

Spitballing, but something like

```
the MinIO Server URL *must* be a resolveable and reachable Fully Qualified Domain Name (FQDN) for the host machine
```

Or something.

Reply:
:thinking: if its not *strictly* required, perhaps we can soften to "Recommend FQDNs to avoid potential connectivity issues"

Reply:
The entire _not really_ conversation turns out to have been a typo. FQDN is required. Editing to say that and include a link.